tests/sentry/replays/test_organization_replay_count.py

```diff
@@ -38,6 +38,13 @@ def setUp(self):
         )
         self.features = {"organizations:session-replay": True}
 
+        # Only projects which have replays will be queried.
+        self.project.flags.has_replays = True
```

Comment:
this won't work as this will break backend error counts. (unless this has been addressed in a separate PR)

Reply:
Are backend errors with to the replays table with a backend project-id?

Reply:
yes. https://github.com/getsentry/sentry/blob/921c82534d5088145a5b4385c9433ef038d16d89/src/sentry/replays/lib/event_linking.py#L133

we could create another flag "has replay_backend_errors" or something.